### DISCUSSION TITLE
Default setting for inform previous parent feature

### DISCUSSION THREAD

**Discussion by ndyck14** [openthread/openthread#8965] [General]
What's the rationale for setting this to 0 by default? Perhaps optimizing messages sent by SED given the most likely reason for parent switch is due to parent gone? Although it seems the consequence of not updating previous parent is much more dire for incoming messages?

https://github.com/openthread/openthread/blob/b8335e0772e9e3557aef693e8fef02ab297a3c26/src/core/config/mle.h#L272

**Comment by jwhui** [comment]
This feature is currently disabled by default because it is not currently covered by the Thread Specification.

See [SPEC-1118](https://threadgroup.atlassian.net/browse/SPEC-1118) for the discussion.

**Reply by jwhui** [reply]
I just submitted https://github.com/openthread/openthread/pull/8969 to enable this by default.

**Reply by EskoDijk** [reply]
My SPEC-1118 conclusion is that it was already covered by the spec (though rather implicitly as a SHOULD requirement without telling the details on how it could be done).

Noting that the feature has a +280 on code size which seems rather large (maybe that could be a reason not to enable it :) )
